.github/workflows/build-and-test.yml

```diff
@@ -34,25 +40,17 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: cargo build
-        run: cargo build --release
-      - name: meson test
-        run: |
-          .github/workflows/test.sh
-          cp ${{ github.workspace }}/build/meson-logs/testlog.txt \
-             ${{ github.workspace }}/build/meson-logs/testlog-$(uname -m).txt
-      - name: cargo build for i686 (assembly temporarily disabled)
+      - name: cargo build for ${{ matrix.target }} (assembly temporarily disabled)
```

Comment:
Should this still say "assembly temporarily disabled"? Isn't assembly enabled for all targets now?